.clang-format-ignore

```diff
@@ -15,3 +15,6 @@ third-party/*
 
 # maintained at https://github.com/sparkle-project/Sparkle
 macosx/Sparkle.framework/*
+
+# Xcode relative derived data path
+DerivedData/*
```

Comment:
DerivedData is by default in `~/Library/Developer/Xcode/DerivedData`.
You normally don't have a DerivedData in your project folder, and we can't fill `.clang-format-ignore` with rule aliases for every custom setup. Just use `build` if you need a build folder.